shared/message/input/Sensors.proto

```diff
@@ -131,13 +131,13 @@ message Sensors {
     float battery = 9;
 
     /// Cartesian position of the CoM {M} of the robot from the torso {T} in the torso space {t}
-    vec4 rMTt = 10;
+    vec3 rMTt = 10;
 
     /// Inertia tensor measured in particle space and relative to the CoM of the particle
     mat3 inertia_tensor = 11;
 
     /// A set of transforms; one for each servo
```

Comment:
add something to this comment so it's not confusing for ppl - ie specify what the extra 4 are for (it's the cameras and feet right?)